test/distributed/org/apache/cassandra/fuzz/topology/HarryTopologyMixupTest.java

```diff
@@ -144,7 +196,7 @@ public PreCheckResult checkPreconditions(State<Spec> state)
     private static CommandGen<Spec> cqlOperations(Spec spec)
     {
         Command<State<Spec>, Void, ?> insert = new HarryCommand(state -> "Harry Insert" + state.commandNamePostfix(), state -> {
-            spec.harry.insert();
+            spec.history.insert();
```

Comment:
personally I prefer `harry`.  `history` exists in the context of `TopologyMixup` and we don't actually build histories as we are eager... we execute right away... so `history` is kinda a confusing name